Header header

int32[] Status
int32[] Pos
int32[] Vel
int32[] Torque